app/src/components/map/CenterMap.vue

```diff
@@ -112,7 +122,33 @@ export default {
         }
       });
     },
+    overlayCallback(indicatorObject) {
+      this.tooltip = formatLabel(indicatorObject, this);
+    },
   },
   beforeDestroy() {},
 };
 </script>
+<style lang="scss" scoped>
+
+.tooltip {
+  position: relative;
+  font-size: 14px;
+  box-shadow: none !important;
+  background: rgba(0, 0, 0, 0.6);
```

Comment:
Here it would be good to explicitly set a text color, since currently the text is not readable in light mode (Vuetify sets the text to black there). In the current implementation we have the tooltip darkmode-agnostic.